From 36df3575bd7cd4e0fd6434f5d12fd5b92534541d Mon Sep 17 00:00:00 2001
From: liyuanrong <liyuanrong1@huawei.com>
Date: Wed, 2 Mar 2022 19:17:00 +0800
Subject: [PATCH] KubeOS:fix sshd startup failed

Temporarily fix the sshd startup failure caused by absence
of the /var/empty/sshd

Signed-off-by: liyuanrong <liyuanrong1@huawei.com>
---
 scripts/generate.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/scripts/generate.sh b/scripts/generate.sh
index 5a97033..7255fcd 100644
--- a/scripts/generate.sh
+++ b/scripts/generate.sh
@@ -258,6 +258,7 @@ function create_img() {
 
 	init_part system.img4 PERSIST "${TMP_MOUNT_PATH}"
 	mkdir ${TMP_MOUNT_PATH}/{var,etc,etcwork}
+	mkdir -p ${TMP_MOUNT_PATH}/var/empty/sshd
 	umount "${TMP_MOUNT_PATH}"
 
 	losetup -D
-- 
1.8.3.1

